_data/team/roles/project_organization.yml

```
@@ -7,4 +7,5 @@ tasks:
   - Gathering usage and community metrics
 
 current_members:
-  - Jenna Swarthout Goddard
+  - [Jenna Swarthout Goddard](https://github.com/jennaswa)
+  - [Irfan Alibay](https://github.com/IAlibay)
```

```suggestion
  - "[Jenna Swarthout Goddard](https://github.com/jennaswa)"
  - "[Irfan Alibay](https://github.com/IAlibay)"
```
This is where CI is currently choking!